Linux - Can't compile after the replacement of devIL for SDL-image

all my troubles on linux after PR: https://github.com/luciusDXL/TheForceEngine/pull/344:

first:
`git pull --recurse-submodules` didn't pull libsdl2-image-dev from the net, maybe it should be added as an external library in the git repo.

second:
after installing it.
passing all the cmake checks ok, when I compile latter I get this:

```
$ make --jobs=16
[  1%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_A11y/accessibility.cpp.o
[  1%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_A11y/filePathList.cpp.o
[  1%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Archive/gobMemoryArchive.cpp.o
[  2%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Archive/gobArchive.cpp.o
[  2%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Archive/labArchive.cpp.o
[  3%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Archive/archive.cpp.o
[  3%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Archive/zipArchive.cpp.o
[  3%] Building C object CMakeFiles/tfe.dir/TheForceEngine/TFE_Archive/zip/zip.c.o
[  2%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Archive/lfdArchive.cpp.o
[  4%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Asset/assetSystem.cpp.o
[  5%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Asset/fontAsset.cpp.o
[  5%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Asset/dfKeywords.cpp.o
[  5%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Asset/colormapAsset.cpp.o
[  6%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Asset/gifWriter.cpp.o
[  6%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Asset/gameMessages.cpp.o
[  6%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Asset/gmidAsset.cpp.o
[  7%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Asset/imageAsset.cpp.o
[  7%] Building CXX object CMakeFiles/tfe.dir/TheForceEngine/TFE_Asset/levelList.cpp.o
In file included from /home/biel/code/jediSith/TheForceEngine/TheForceEngine/TFE_Asset/imageAsset.cpp:1:
/home/biel/code/jediSith/TheForceEngine/TheForceEngine/TFE_Asset/imageAsset.h:6:10: fatal error: SDL_image.h: El fitxer o directori no existeix
    6 | #include <SDL_image.h>
      |          ^~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/tfe.dir/build.make:300: CMakeFiles/tfe.dir/TheForceEngine/TFE_Asset/imageAsset.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:469: CMakeFiles/tfe.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -54,6 +54,7 @@ if(LINUX)
 	find_package(OpenGL REQUIRED)
 	target_include_directories(tfe PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
 	target_include_directories(tfe PRIVATE ${SDL2_INCLUDE_DIRS})
+	target_include_directories(tfe PRIVATE ${SDL2_IMAGE_INCLUDE_DIRS})
 	target_link_libraries(tfe PRIVATE
 				${OPENGL_LIBRARIES}
 				${GLEW_LIBRARIES}

